_delphi_utils_python/delphi_utils/runner.py

```diff
@@ -67,6 +67,7 @@ def run_indicator_pipeline(indicator_fn:  Callable[[Params], None],
     validator = validator_fn(params)
```

Comment:
The archiver already logs run timing, and each indicator has its own bit of logging which includes that; this adds logging for the flash & validation steps.

Reply:
the individual indicators' run time logging looks mostly sufficient, except in `quidel_covidtest` where it happens [at program exit](https://github.com/cmu-delphi/covidcast-indicators/blob/57f5a15d9c6cb5ccbac9dd9f814045772e93bd19/quidel_covidtest/delphi_quidel_covidtest/run.py#L124), which will include anything that runs after the core indicator function and thus lead to inaccuracy.

you can refactor this so the runner does the timing and logging, with `indicator_fn` (aka each indicator's `run_module()`) returning a dict of metrics it wants logged (like csv_export_count and max_lag_in_days)

Reply:
Hm, I can actually think of one disadvantage of that - it means that the summary line will be gone from the logs if the indicator is ran individually (e.g. `env/bin/python -m delphi_quidel_covidtest` as the [README](https://github.com/cmu-delphi/covidcast-indicators/blob/main/quidel_covidtest/README.md) suggests).
Are we OK with this?

Reply:
a lot of the documentation in this repo is is quite old and should be brought up to date.

is it ever desirable to run an indicator _*without*_ validation and archiving?  perhaps we can answer that in #1895.  you should at least fix the timing for the quidel indicator in the meanwhile.